### PR TITLE
CI: Circle: updates the RiffRaff plugin and artifact upload

### PR DESCRIPTION
**Pull request by rrees**
Switches the build file to just build one project (path-manager) and adds the Circle configuration to upload an artifact.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -5,3 +8,13 @@ dependencies:
     - "target/streams"
     - "project/target/resolution-cache"
     - "project/target/streams"
+  override:
+    - sbt test:compile
+test:
+  override:
+    - sbt test
+  post:
+    - sbt riffRaffUpload
```
Comment (sihil) [inline circle.yml]: IIRC `post` actions in the `test` section happen regardless of the test success or failure (resulting is builds being available and continuously deployed in RiffRaff that failed their tests). For flex I solved this by doing the upload in the deploy section. See https://github.com/guardian/flexible-content/blob/master/circle.yml#L69-L73

Comment (rrees) [inline circle.yml]: The project doesn't yet have tests so I wouldn't have noticed.
I will go back and apply this change to the other CircleCI projects I have done as a job lot.

Comment (rrees) [inline circle.yml]: I "circle" back on it :laughing:

**Comment by rrees** [comment]
@sihil Fully circled

**Pull request merged**